zebra-utils/Cargo.toml

```diff
@@ -120,8 +120,9 @@ tokio = { version = "1.37.0", features = ["full"], optional = true }
 
 jsonrpc = { version = "0.18.0", optional = true }
 
-zcash_primitives = { version = "0.13.0", optional = true }
-zcash_client_backend = {version = "0.10.0-rc.1", optional = true}
+zcash_primitives = { version = "0.15.0", optional = true }
```

Comment:
I am getting a mismatch if i leave this as 0.14.0:

```
error[E0308]: mismatched types
   --> zebra-utils/src/bin/scanning-results-reader/main.rs:75:72
    |
75  |                     decrypt_transaction(&network, block_number.into(), &tx, &ufvk_with_acc_id)
    |                     -------------------                                ^^^ expected `zcash_primitives::transaction::Transaction`, found a different `zcash_primitives::transaction::Transaction`
    |                     |
    |                     arguments to this function are incorrect
    |
    = note: `zcash_primitives::transaction::Transaction` and `zcash_primitives::transaction::Transaction` have similar names, but are actually distinct types
note: `zcash_primitives::transaction::Transaction` is defined in crate `zcash_primitives`
   --> /home/alfredo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/zcash_primitives-0.14.0/src/transaction/mod.rs:303:1
    |
303 | pub struct Transaction {
    | ^^^^^^^^^^^^^^^^^^^^^^
note: `zcash_primitives::transaction::Transaction` is defined in crate `zcash_primitives`
   --> /home/alfredo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/zcash_primitives-0.15.0/src/transaction/mod.rs:303:1
    |
303 | pub struct Transaction {
    | ^^^^^^^^^^^^^^^^^^^^^^
    = note: perhaps two different versions of crate `zcash_primitives` are being used?
note: function defined here
   --> /home/alfredo/.cargo/registry/src/index.crates.io-6f17d22bba15001f/zcash_client_backend-0.12.1/src/decrypt.rs:103:8
    |
103 | pub fn decrypt_transaction<'a, P: consensus::Parameters, AccountId: Copy>(
```

For now i am leaving it like that and adding an entry in deny.toml.